test/unit/theory/theory_ff_groebner_black.cpp

```diff
@@ -0,0 +1,204 @@
+/******************************************************************************
+ * Top contributors (to current version):
+ *   Alex Ozdemir
+ *
+ * This file is part of the cvc5 project.
+ *
+ * Copyright (c) 2009-2022 by the authors listed in the file AUTHORS
+ * in the top-level source directory and their institutional affiliations.
+ * All rights reserved.  See the file COPYING in the top-level source
+ * directory for licensing information.
+ * ****************************************************************************
+ *
+ * Black box testing of ff incremental groebner basis computation.
+ */
+
+#ifdef CVC5_USE_COCOA
+#include <CoCoA/BigInt.H>
+#include <CoCoA/QuotientRing.H>
+#include <CoCoA/RingZZ.H>
+#include <CoCoA/SparsePolyOps-ideal.H>
+#include <CoCoA/SparsePolyRing.H>
+#include <CoCoA/ring.H>
+#include <CoCoA/symbol.H>
+#endif  // CVC5_USE_COCOA
```

Comment:
Like the other unit tests, let's just guard the whole file.

Reply:
Whoops. Good catch.